src/tokens/transformation/skia/templates/colors.h.template

```diff
@@ -16,3 +16,27 @@ namespace leo::dark {
   return `constexpr SkColor ${prop.name} = SkColorSetRGB(${prop.value})`;
 }).map(s => `${s};`).join('\n') %>
 } // namespace leo::dark
+
+namespace leo {
+
+enum Theme {
+  kLight,
+  kDark
+};
+
+enum Color {
```

Comment:
How about using E_CPONLY(), which is used by brave_color_ids.h.

Reply:
Hmm, that looks interesting - it says it provides a string name for the color, what exactly is the purpose? Is it for when you enable devtools for views?

I think I'd feel (marginally) better leaving it out for now, and coming back and adding it when we need it. What do you think?

(I haven't encountered it before, I'm happy to add it if you think it'll be useful though)